install/syndesis.yml

```diff
@@ -2409,6 +2409,7 @@ objects:
     resources:
     - deploymentconfigs
     - deploymentconfigs/scale
+    - deploymentconfigs/finalizers
```

Comment:
This file can be deleted

Reply:
Think we need a proper chore / cleanup commit which addresses these extra files. Will simply revert change for the moment.